src/app/_services/api/accounts/consent.service.ts

```diff
@@ -6,43 +6,43 @@ import {Observable} from 'rxjs';
 import {UserConsent} from '@app/_models/user_consent';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root'
 })
 export class ConsentService {
-  private consentAPIUrl = new URL('/api/user-consent/', environment.apiBaseUrl).toString();
+    private consentAPIUrl = new URL('/api/user-consent/', environment.apiBaseUrl).toString();
 
-  constructor(private http: HttpClient) {
-  }
+    constructor(private http: HttpClient) {
+    }
 
-  PostConsent(input: any) {
-    return this.http.post(this.consentAPIUrl, input, {responseType: 'text'}).pipe(
-      map((response) => {
-          if (response) {
-            return response;
-          }
-        },
-        (error: any) => {
-          return error;
-        }
-      )
-    );
-  }
+    postConsent(input: any) {
```

Comment:
Are postConsent and withdrawConsent exactly the same? We probably only need one of them and the input determines if the user is consenting or withdrawing the consent.

Reply:
Yes they are I'll fix it.